packages/terminology/src/operations/conceptMapImport.ts

```diff
@@ -0,0 +1,118 @@
+import { FhirRequest, FhirResponse } from "@medplum/fhir-router";
+import { ConceptMap } from "@medplum/fhirtypes";
+import { normalizeOperationOutcome } from "@medplum/core";
+import { v4 as uuidv4 } from "uuid";
+
+import { getSqliteClient } from "../sqlite";
+
+function generateReverseConceptMaps(conceptMap: ConceptMap): ConceptMap[] {
```

Comment:
the idea here is to create bidirectional concept maps. 
If we know a => b, then why not also put b => a into the term server too.